README.md

```diff
@@ -1 +1,47 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello Microcerse
+
+This is a simple "Hello Microverse"  message for linters practice
+
+
+## Built With
+
+- HTML & CSS
+- Linters
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+A code compiler like Visual Studio Code is needed
+
+### Setup
+
+Git clone can be used to copy the codes from the repository or download ZIP
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@dieum512](https://github.com/dieum512/)
+- Twitter: [@dieumakonkw](https://twitter.com/dieumakonkw)
+- LinkedIn: [Dieum Akonkwa](https://www.linkedin.com/in/dieum-akonkwa-7981b924b/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
```

Comment:
- [ ] this is an old readme template, as part of the requirements kindly update your readme file using the new microverse [README ](https://github.com/microverseinc/readme-template) template 🙏